spec/unit/snowplow_spec.lua

```diff
@@ -61,13 +47,40 @@ describe("snowplow", function()
   -- --------------------------------------------------------------
   -- Verify constructed tracker tables
 
-  it("new_tracker_for_uri() should correctly create a tracker", function()
-    local t = snowplow.new_tracker_for_uri("c.snplow.com")
-    assert_tracker(t, "http://c.snplow.com/i")
+  it("new_tracker() should correctly create a tracker", function()
+    local t = snowplow.new_tracker("c.snplow.com")
```

Comment:
Sorry if I missed it, but I don't see a spec where we check that an endpoint already containing protocol at the start of the URI is handled correctly. Can you add this just to cover all variations?

Reply:
Nevermind, I see that it is the last test in this file 🤦‍♂️